src/main/java/org/sefglobal/scholarx/model/SentEmails.java

```diff
@@ -0,0 +1,84 @@
+package org.sefglobal.scholarx.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.ManyToOne;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.sefglobal.scholarx.util.ProgramState;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+ 
+@Entity
+@Table(name = "email")
+@JsonIgnoreProperties({"createdAt"})
+public class SentEmails extends BaseScholarxModel {
```

Comment:
```suggestion
public class SentEmail extends BaseScholarxModel {
```

Reply:
Fixed it with commit d8de1214775427ed2572e95fd6f81f526778a504